frontend/components/incompleteupload/Editor.vue

```diff
@@ -1,48 +1,58 @@
 <script setup lang="ts">
-import { type EditorIncompleteUpload } from '@/lib/editor'
+import {
+  type EditorIncompleteUploadFields as EditorFields,
+  type EditorIncompleteUploadValues as EditorValues,
+} from '@/lib/editor'
+
+const prefix = 'components/incompleteupload/Editor'
+
+const { t } = useI18n()
+const tt = (key: string) => t(`${prefix}.${key}`)
 
 interface Props {
-  editorIncompleteUpload: EditorIncompleteUpload
+  editorValues: EditorValues
+  editorFields: EditorFields
 }
 interface Emits {
-  (e: 'update:editorIncompleteUpload', ei: EditorIncompleteUpload): void
+  (e: 'update:editorValues', evs: EditorValues): void
 }
 const props = defineProps<Props>()
 const emit = defineEmits<Emits>()
 
-const model = computed({
-  get: () => props.editorIncompleteUpload,
-  set: (editorIncompleteUpload: EditorIncompleteUpload) => { emit('update:editorIncompleteUpload', editorIncompleteUpload) },
+const efs = computed(() => props.editorFields)
```

Comment:
Why wrap this instead of just using `props.editorFields` directly? To make it read-only? And elsewhere

Reply:
Exactly - readonly :) Added a comment for that.

Reply:
Just for readability here and elsewhere - found it easier to grok and copy/paste code.